validator/src/journal/chain_ffi.rs

```diff
@@ -520,7 +520,7 @@ impl BlockStore for PyBlockStore {
                 .and_then(|blkw| blkw.getattr(py, "block"))
                 .map_err(|py_err| {
                     pylogger::exception(py, "Unable to call block_store.get_blocks", py_err);
-                    BlockStoreError::Error(format!("Unable to get blocks"))
+                    BlockStoreError::Error("Unable to get blocks".into())
```

Comment:
Again, we have a third way of `&str` to `String`, here. Let's pick one.

Reply:
I went with .into() for changing an &str into String.